rasa/cli/data.py

```diff
@@ -480,11 +495,37 @@ def _assert_two_stage_fallack_policy_is_migratable(config: Dict):
             f"'{USER_INTENT_OUT_OF_SCOPE}' before migrating the model configuration. "
         )
 
+    if (
+        two_stage_fallback_config.get(
+            "fallback_nlu_action_name", ACTION_DEFAULT_FALLBACK_NAME
+        )
+        != ACTION_DEFAULT_FALLBACK_NAME
+    ):
+        rasa.shared.utils.cli.print_error_and_exit(
+            f"The TwoStageFallback in Rasa Open Source 2.0 has to use the action "
+            f"'{ACTION_DEFAULT_FALLBACK_NAME}' for cases when the user denies the "
+            f"suggestion multiple times. "
+            f"Please change the parameter 'fallback_nlu_action_name' to "
+            f"'{ACTION_DEFAULT_FALLBACK_NAME}' before migrating the model "
+            f"configuration. "
+        )
+
+
+def _assert_only_one_fallback_policy_present(policies: List[Text]) -> None:
```

Comment:
is there a test for this sys exit?

Reply:
`test_convert_config_with_two_stage_fallback_policy`

Reply:
`test_convert_config_with_invalid_policy_config`